sdkconfig-esp32s3-devkitc-n8

```diff
@@ -455,17 +455,17 @@ CONFIG_ESPTOOLPY_MONITOR_BAUD=115200
 # CONFIG_PARTITION_TABLE_SINGLE_APP_LARGE is not set
 # CONFIG_PARTITION_TABLE_TWO_OTA is not set
 CONFIG_PARTITION_TABLE_CUSTOM=y
-CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="partitions-4m.csv"
-CONFIG_PARTITION_TABLE_FILENAME="partitions-4m.csv"
+CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="partitions-8m.csv"
+CONFIG_PARTITION_TABLE_FILENAME="partitions-8m.csv"
 CONFIG_PARTITION_TABLE_OFFSET=0x8000
 CONFIG_PARTITION_TABLE_MD5=y
 # end of Partition Table
 
 #
 # Compiler options
 #
-CONFIG_COMPILER_OPTIMIZATION_DEFAULT=y
-# CONFIG_COMPILER_OPTIMIZATION_SIZE is not set
+# CONFIG_COMPILER_OPTIMIZATION_DEFAULT is not set
+CONFIG_COMPILER_OPTIMIZATION_SIZE=y
```

Comment:
Na ESP32 už tohle bylo na `-Os`. Že by tam bylo nějaké undefined behavior které se chová jinak na `-Og` a `-Os` a proto to na S3 fungovalo lépe?